src/main.rs

```diff
@@ -65,7 +65,8 @@ fn main() {
         (
             module: services::packet_processor::start_inbound,
             name: inbound_packet_processor,
-            dependencies: [messenger, player_state, block_state, patchwork_state]
+            dependencies: [messenger, player_state, block_state, patchwork_state],
+            extras: [None]
```

Comment:
I don't think extras is the right name for this. I don't know what the right name is because I'm not entirely sure what it's for, but if this is something we want to include in the services API we should give it a name that matches that particular use.